README.md

```diff
@@ -1,19 +1,186 @@
 # How To Write More Correct Software
 
 Welcome to the How To Write More Correct Software workshop!
+You can skip right to [the workshop](./instructions/steps.md) if you like.
 
-We're here to help you learn how to use two powerful tools
-to make more of your code more correct:
+This content is currently a work in progress - use at your own risk, mind our dust, and stay tuned!
 
-1. [Dafny](http://dafny.org/), a programming language that formally verifies your implementation matches your specification
-2. [Duvet](https://github.com/awslabs/duvet), a code quality tool for measuring how well your human-readable specification is covered in code
+## Why does code have to be correct?
 
-This content is currently a work in progress - use at your own risk, mind our dust, and stay tuned!
+If code is not correct it may not work.
+In which case you risk blocked builds.
+What if it is incorrect but everything still works?
+This can be much worse.
+
+Let's keep it positive.
+Correct software improves
+
+* Operations -
+  Your software performs as you expect, more uptime.
+* Evolution -
+  You know what needs to change, faster development.
+* Security -
+  You can reason about security properties, better sleep.
```

Comment:
I'm inclined to leave it.
Since the point is exactly that this "correctness is important"
is not always so obvious.
Empirically, since everything is not formally verified with tests and a specification :)